src/blockchain/node/manager/Manager.cpp

```diff
@@ -1374,5 +1376,6 @@ auto Base::Wallet() const noexcept -> const node::Wallet& { return wallet_; }
 Base::~Base()
 {
     protect_shutdown([this] { shut_down(); });
+    shutdown_timers();
```

Comment:
shut_down() may, and should be called earlier, so please move thistimers shutdown to inside shut_down()